filebeat/crawler/prospector_log.go

```diff
@@ -38,7 +38,23 @@ func NewProspectorLog(p *Prospector) (*ProspectorLog, error) {
 
 func (p *ProspectorLog) Init() {
 	logp.Debug("prospector", "exclude_files: %s", p.config.ExcludeFiles)
-	p.scan()
+
+	logp.Info("Load previous states from registry into memory")
+
+	for path, fileinfo := range p.getFiles() {
```

Comment:
Starts from the beginning. See also discussion here: https://github.com/elastic/beats/issues/1022#issuecomment-208193158